tests/functional/strategy/test_misc.py

```diff
@@ -81,6 +81,39 @@ def test_harvest_tend_trigger(chain, gov, vault, token, TestStrategy):
     assert strategy.estimatedTotalAssets() == 0
 
 
+def test_sweep_authority(
```

Comment:
This test is for the "authority" modifier, although the test is not correctly picked up by brownie due to https://github.com/eth-brownie/brownie/issues/809